### ISSUE TITLE
C3 runtime: plugin action never ends if its js function returns true

### ISSUE THREAD

**Issue by valerypopoff** [Scirra/Construct-3-bugs#2108]
## Problem description
If a function that describes a plugin's action returns "true":
```
	C3.Plugins.MyCompany_SingleGlobal.Acts =
	{
		Alert()
		{
			alert("Test property = " + this._testProperty);
			return true;
		}
	};
```
the action works fine but next actions in the event sheet never get called:
![img](https://user-images.githubusercontent.com/5064630/46569913-ffc7d900-c964-11e8-8db9-759e4326a25d.png)

The problem only reproduces in C3 runtime. C2 runtime works fine. It reproduces with the sample plugin I attached. It reproduces with my other plugin. It also reproduces if you add "return true" to any other plugin's code. For example, I tried it with official Browser's ConsoleLog:
```
	ConsoleLog(a,b)
	{
		b = b.toString(),0===a?console.log(b):1===a?console.warn(b):2==a&&console.error(b);
		return true;
	}
```

I need my ACEs functions to be able to return "true" or any other values because I reuse them by calling them from other ACEs. For example, my JS Plugin (https://www.construct.net/en/make-games/addons/1/javascript) is now broken in C3 runtime because of this.

## Attach a .c3p
[bug.zip](https://github.com/Scirra/Construct-3-bugs/files/2452906/bug.zip)

And here's the sample plugin used in this project. I downloaded it from https://www.construct.net/en/make-games/manuals/addon-sdk and simply added "return true" to the action's function:
[singleGlobalPlugin.zip](https://github.com/Scirra/Construct-3-bugs/files/2452907/singleGlobalPlugin.zip)

## Steps to reproduce
1. Install the plugin
2. Switch to C3 runtime
3. Run the project
4. See the browser alert that says "Test property = 0"
5. Click Ok on the alert
6. Look in the console

## Observed result
"FINISHED" is not logged to the console

## Expected result
"FINISHED" is logged to the console

## Affected browsers
* Chrome: yes

## System details
<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 69.0.3497.100
Browser engine: Blink
Browser architecture: 64-bit
Context: browser
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Safari/537.36
C3 release: r120 (stable)
Language setting: en-US

Local storage
Storage quota (approx): 30 gb
Storage usage (approx): 332 mb (1.1%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: no
Vendor: Google Inc.
Renderer: ANGLE (NVIDIA GeForce GTX 1080 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions:

EXT_color_buffer_float
EXT_texture_filter_anisotropic
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by valerypopoff** [comment]
OH. MY. GOD. Thanks a bundle(

**Comment by valerypopoff** [comment]
Is just "return" alright?

**Comment by AshleyScirra** [comment]
Yes, that is the same as returning undefined, which is what a function with no return statement returns.
